stdlib/runtime/build.gradle

```diff
@@ -1,22 +1,27 @@
 apply from: "$rootDir/gradle/balNativeLibProject.gradle"
 
 dependencies {
-    implementation project(':ballerina-lang')
     implementation project(':ballerina-core')
+    implementation project(':ballerina-lang')
     implementation project(':ballerina-builtin')
-    testCompile 'org.testng:testng:6.13.1'
     testCompile project(':ballerina-launcher')
+    testCompile 'org.slf4j:slf4j-jdk14:1.7.22'
+    testCompile 'org.testng:testng:6.13.1'
 
     baloImplementation project(path: ':ballerina-builtin', configuration: 'baloImplementation')
 }
 
-description = 'Ballerina - Runtime'
+task copyGeneratedBalo(type: org.gradle.api.tasks.Copy) {
+    from "$buildDir/generated-balo/repo/ballerina"
+    into "$buildDir/lib/repo/ballerina"
+}
 
 test {
-    // TODO: Fix tests and remove test skip block
-    exclude '**/*'
-    
+    dependsOn copyGeneratedBalo
     useTestNG() {
         suites 'src/test/resources/testng.xml'
     }
-}
\ No newline at end of file
+    systemProperty "ballerina.home", "$buildDir"
```

Comment:
@daneshk Have moved this line to [balNativeLibProject plugin.](https://github.com/ballerina-platform/ballerina-lang/pull/14082/files) so we don't need this line anymore